meteor_packages/mats-common/imports/startup/api/matsMethods.js

```diff
@@ -1632,27 +1632,31 @@ const resetApp = async function (appRef) {
         const appPools = appRef.appPools;
         const type = appRef.appType;
         const dbType = appRef.dbType ? appRef.dbType : matsTypes.DbTypes.mysql;
-        const appName = appRef.app;
-        const appTitle = appRef.title;
-        const appGroup = appRef.group;
-        var color;
+        const appName = Meteor.settings.public.app ? Meteor.settings.public.app : "unnamed";
+        const appTitle = Meteor.settings.public.title ? Meteor.settings.public.title : "Unnamed App";
+        const appGroup = Meteor.settings.public.group ? Meteor.settings.public.group : "Misc. Apps";
+        var appDefaultGroup = "";
+        var appDefaultDB = "";
+        var appDefaultModel = "";
+        var appColor;
         switch (type) {
             case matsTypes.AppTypes.mats:
-                if (dbType == matsTypes.DbTypes.couchbase) {
-                    color = "#33abbb";
+                if (dbType === matsTypes.DbTypes.couchbase) {
+                    appColor = "#33abbb";
                 } else {
-                    // default to mysql so that old apps won't break
-                    color = "#3366bb";
+                    appColor = Meteor.settings.public.color ? Meteor.settings.public.color : "#3366bb";
```

Comment:
Two questions:

1. Am I understanding correctly that this would hardwire the color for couchbase apps?

    If so, it looks like this would ignore whatever color the user was requesting in the settings file which could be confusing. Would the following lead to less user confusion?

    ```js
    if (dbType === matsTypes.DbTypes.couchbase) {
      appColor = Meteor.settings.public.color ? Meteor.settings.public.color : "#33abbb";
    } else {
      appColor = Meteor.settings.public.color ? Meteor.settings.public.color : "#3366bb";
    }
    ```

2. Does the code need to set `appDefaultGroup`, `appDefaultDB`, and `appDefaultModel` in the `AppTypes.mats` switch statement here as well, or are those settings METexpress-specific?